hedgehog/test/Test/Hedgehog/Seed.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE TemplateHaskell #-}
+module Test.Hedgehog.Seed (
+    tests
+  ) where
+
+import           Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Internal.Seed as Seed
+
+data Assert =
+  Assert {
+      expected :: !Seed
+    , actual   :: !Seed
+    } deriving (Show)
+
+-- | Verify that SplitMix avoids pathological γ-values, as discussed by
+--   Melissa E. O'Neill in the post with title Bugs in SplitMix(es), at
+--   http://www.pcg-random.org/posts/bugs-in-splitmix.html
+--
+--   See also:
+--   https://github.com/hedgehogqa/haskell-hedgehog/issues/191
+--
+prop_avoid_pathological_gamma_values :: Property
+prop_avoid_pathological_gamma_values =
```

Comment:
This looks fully deterministic to me, is that the case?

If so, we don't need to run this 100 times.

Suggest something like 

```haskell
withTests 1 . property $ do
  for asserts $ \a ->
    expected a === actual a
```

Reply:
Indeed, those are deterministic, so just doing the comparison once is enough. Good catch 👍 Fixed and force-pushed.